packages/esbuild-plugin/src/index.js

```diff
@@ -115,6 +115,7 @@ export default function stylexPlugin({
               jsxSyntaxPlugin,
               stylexBabelPlugin.withOptions({
                 ...options,
+                treeshakeCompensation: true,
```

Comment:
Ideally, you should make `treeshakeCompensation` an optional argument that defaults to `true`.

Reply:
I think moving it above `...options` should suffice?

Reply:
Yeah this should work.